examples/src/app/page.tsx

```diff
@@ -10,10 +10,26 @@ export default async function Home({ searchParams }: HomeProps) {
     <div className="space-y-4 p-4 lg:p-8">
       <Title>Demos</Title>
       <div className="grid grid-cols-1 lg:grid-cols-2 gap-4 max-w-2xl">
-        <ProjectLink name="Color Grid" url="/color-grid" />
-        <ProjectLink name="To Do List" url="/to-do-list" />
-        <ProjectLink name="Text Editor" url="/text-editor" />
-        <ProjectLink name="Code Editor" url="/code-editor" />
+        <ProjectLink
+          name="Color Grid"
+          url="/color-grid"
+          description="Drop different shades of pink into a grid."
+        />
+        <ProjectLink
+          name="To Do List"
+          url="/to-do-list"
+          description="Write and clear a to do list."
+        />
+        <ProjectLink
+          name="Text Editor"
+          url="/text-editor"
+          description="A collaborative text editor that uses a quill yjs binding."
```

Comment:
What do you think about: "A collaborative text editor built on top of the open-source Quill package." ?